dev-meetings/2025/2025-08-29/README.md

```diff
@@ -1,9 +1,95 @@
 # Gammapy Developer Meeting 
- * Friday, August 29, 2025, at 2 pm (CET) 
+ * Friday, August 29, 2025, at 2 pm (CET)  approx. 14:05-15:17
  * Gammapy Developer Meeting on Zoom (direct link on Slack) 
+
+Attendees: Leander Schlegel (LS), Tomas Bylund (TB), Aritra Gupta (AG), Régis Terrier (RT), Daniel Morcuende (DM), Kirsty Feijen (KF), Atreyee Sinha (AS), Mattias Fuessling (MF)
+ 
 # Agenda
 ## General information
 
+### v2.0 release status
+RT reports: 
+Release made last Tuesday (August 26), AS made most of the work. 
+
+Overall went fine. Validation and benchmarks scripts used for the released version were also tagged. 
+
+The instructions for making the final full release did not contain the step where `dev/prepare-release.py` is run, and consequently some metadata for the release is incomplete.
+
+The automatic adding of the new version to Zenodo failed, so will have to be done manually.
+Bruno has done this before and back then contacted Zenodo team for this.
+
+RT: Has someone already installed and checked that everything is okay with release?
+
+DM confirms for mamba.
+
+RT: All steps are now checked and we should announce the new version. 
+
+AS: Should make sure mails are sent to VERITAS, MAGIC, HAWC, SWGO, need to find people who can send to their mailing lists. 
+
+RT: Have to prepare text for announcement mails. After mails are sent, there should be celebrations.
+
+### Reflections on the new release
+RT: It was a long time to make this release, probably the roadmap was too ambitious.
+
+AS: Gabriel Emery was missed for the contributor list somehow, but he is in the author list. Unclear how this happened. Is it worth to update the docs to ensure he is included?
+RT: Probably not (now ???), rebuilding the docs is complicated, as it will be fixed soon probably in any case.
```

Comment:
```suggestion
RT: Probably not before next bugfix as manually rebuilding the docs is complicated, as it will be fixed soon probably in any case.
```